csrc/scheduler/registry.cpp

```diff
@@ -90,6 +89,15 @@ bool checkCanSchedule(Fusion* fusion, SchedulerType scheduler_type) {
     return false;
   }
 
+  // check PreprocessGroupedMatmulInputSf's output is in global memory by
+  // forcing it as a fusion segment output
```

Comment:
@naoyam I think I should have done a memory promotion instead of forcing a segment. But since the consumer is in a separate kernel anyway, I don't think the two alternative has any meaningful difference.